_docs/_latest/configure/notifier/index.md

```diff
@@ -171,6 +171,36 @@ notifier:
             webhook_url: https://hooks.slack.com/services/foobar
 ```
 
+### CSCC Notification
+
+Forseti Security can configured to send violations to [Cloud Security Command Center (Cloud SCC)](https://cloud.google.com/security-command-center/).
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+1. Navigate to the `notifier` > `violation` > `cscc` section.
+
+The options below are available for you to configure:
+
+* `enabled:`
+  * **Description**: Whether to send notification to Cloud SCC.
+  * **Valid values**: one of valid `true` or `false`
+
+* `mode:`
+  * **Description**: How to send the violations to Cloud SCC.
+  * **Valid values**: one of valid `api` or `bucket`
+  * **Note**: `api` mode will only work if Forseti is using a service account
+from the Cloud Security Center project.
+
+* `organization_id:`
+  * **Description**: The organization id.
```

Comment:
Can we also add something like "used only when model is api" and same for the gcs bucket description.

Reply:
Done